From ca646eb3e5bd4612900ef7a7ed70b7e4de73ccf7 Mon Sep 17 00:00:00 2001
From: Atticus McIlraith <gamingisscience@gmail.com>
Date: Thu, 16 Sep 2021 10:26:06 -0500
Subject: [PATCH] Minor Map QOL Fixes

Added nanomed to gaunt, fixed piping in gaunt/garuda, maxed default pressure to hangar on both valves, fixed SD setup. Relocated locker.
---
 maps/torch/torch1_deck5.dmm  | 73 +++++++++++++++++++++--------
 maps/torch/torch2_deck4.dmm  | 22 ++++-----
 maps/torch/torch3_deck3.dmm  | 90 ++++++++++++++++++------------------
 maps/torch/torch4_deck2.dmm  |  4 +-
 maps/torch/torch5_deck1.dmm  |  7 +--
 maps/torch/torch6_bridge.dmm | 17 ++++++-
 6 files changed, 126 insertions(+), 87 deletions(-)

diff --git a/maps/torch/torch1_deck5.dmm b/maps/torch/torch1_deck5.dmm
index c6412869e2..e90492c019 100644
--- a/maps/torch/torch1_deck5.dmm
+++ b/maps/torch/torch1_deck5.dmm
@@ -1151,8 +1151,8 @@
 	icon_state = "intact"
 	},
 /obj/effect/floor_decal/industrial/warning,
-/obj/machinery/atmospherics/valve/open{
-	open = 0
+/obj/machinery/atmospherics/binary/passive_gate{
+	dir = 1
 	},
 /turf/simulated/floor/tiled/techfloor/grid,
 /area/guppy_hangar/start)
@@ -2003,6 +2003,13 @@
 	dir = 1;
 	icon_state = "handrail"
 	},
+/obj/machinery/vending/wallmed1{
+	dir = 1;
+	icon_state = "wallmed";
+	name = "Emergency NanoMed";
+	pixel_x = 0;
+	pixel_y = -32
+	},
 /turf/simulated/floor/tiled,
 /area/exploration_shuttle/airlock)
 "dC" = (
@@ -2288,15 +2295,13 @@
 /area/guppy_hangar/start)
 "eg" = (
 /obj/effect/floor_decal/industrial/outline/orange,
-/obj/machinery/portable_atmospherics/canister/carbon_dioxide/engine_setup{
-	start_pressure = 7498
-	},
 /obj/machinery/atmospherics/portables_connector{
 	dir = 4
 	},
 /obj/machinery/light/small{
 	dir = 8
 	},
+/obj/machinery/portable_atmospherics/canister/carbon_dioxide,
 /turf/simulated/floor/plating,
 /area/exploration_shuttle/atmos)
 "eh" = (
@@ -2305,7 +2310,7 @@
 	d2 = 4;
 	icon_state = "1-4"
 	},
-/obj/machinery/atmospherics/binary/pump{
+/obj/machinery/atmospherics/binary/pump/high_power{
 	dir = 4
 	},
 /turf/simulated/floor/tiled/techfloor/grid,
@@ -2435,7 +2440,7 @@
 	dir = 4
 	},
 /obj/random_multi/single_item/runtime,
-/obj/machinery/atmospherics/binary/pump,
+/obj/machinery/atmospherics/binary/pump/high_power,
 /turf/simulated/floor/tiled/techfloor/grid,
 /area/exploration_shuttle/atmos)
 "et" = (
@@ -2882,8 +2887,8 @@
 /area/exploration_shuttle/power)
 "fx" = (
 /obj/machinery/power/apc/shuttle/charon{
-	dir = 4;
-	icon_state = "apc0"
+	dir = 1;
+	pixel_y = 15
 	},
 /obj/structure/cable/cyan{
 	d2 = 8;
@@ -3072,6 +3077,39 @@
 /obj/item/weapon/storage/mre/random,
 /obj/random/single/cola,
 /obj/random/single/cola,
+/obj/item/weapon/storage/mre/random,
+/obj/item/weapon/storage/mre/random,
+/obj/item/weapon/storage/mre/random,
+/obj/item/weapon/storage/mre/random,
+/obj/item/weapon/storage/mre/random,
+/obj/item/weapon/storage/mre/random,
+/obj/item/weapon/storage/mre/random,
+/obj/item/weapon/storage/mre/random,
+/obj/item/weapon/storage/mre/random,
+/obj/item/weapon/storage/mre/random,
+/obj/item/weapon/storage/mre/random,
+/obj/item/weapon/storage/mre/random,
+/obj/item/weapon/storage/mre/random,
+/obj/item/weapon/storage/mre/random,
+/obj/item/weapon/storage/mre/random,
+/obj/item/weapon/storage/mre/random,
+/obj/item/weapon/storage/mre/random,
+/obj/item/weapon/storage/mre/random,
+/obj/item/weapon/reagent_containers/food/drinks/cans/waterbottle,
+/obj/item/weapon/reagent_containers/food/drinks/cans/waterbottle,
+/obj/item/weapon/reagent_containers/food/drinks/cans/waterbottle,
+/obj/item/weapon/reagent_containers/food/drinks/cans/waterbottle,
+/obj/item/weapon/reagent_containers/food/drinks/cans/waterbottle,
+/obj/item/weapon/reagent_containers/food/drinks/cans/waterbottle,
+/obj/item/weapon/reagent_containers/food/drinks/cans/waterbottle,
+/obj/item/weapon/reagent_containers/food/drinks/cans/waterbottle,
+/obj/item/weapon/reagent_containers/food/drinks/cans/waterbottle,
+/obj/item/weapon/reagent_containers/food/drinks/cans/waterbottle,
+/obj/item/weapon/reagent_containers/food/drinks/cans/waterbottle,
+/obj/item/weapon/reagent_containers/food/drinks/cans/waterbottle,
+/obj/item/weapon/reagent_containers/food/drinks/cans/waterbottle,
+/obj/item/weapon/reagent_containers/food/drinks/cans/waterbottle,
+/obj/item/weapon/reagent_containers/food/drinks/cans/waterbottle,
 /turf/simulated/floor/tiled/techfloor/grid,
 /area/exploration_shuttle/cargo)
 "fS" = (
@@ -5226,7 +5264,7 @@
 "le" = (
 /obj/machinery/power/apc/shuttle/charon{
 	dir = 4;
-	icon_state = "apc0"
+	pixel_x = 25
 	},
 /obj/machinery/atmospherics/unary/vent_scrubber/on{
 	dir = 1
@@ -5421,14 +5459,12 @@
 /turf/simulated/floor/tiled/dark/monotile,
 /area/quartermaster/expedition/storage)
 "lz" = (
-/obj/machinery/portable_atmospherics/canister/carbon_dioxide/engine_setup{
-	start_pressure = 4769
-	},
 /obj/machinery/light/small{
 	dir = 4;
 	icon_state = "bulb1"
 	},
 /obj/effect/floor_decal/industrial/outline/orange,
+/obj/machinery/portable_atmospherics/canister/carbon_dioxide,
 /turf/simulated/floor/tiled/dark,
 /area/quartermaster/expedition/atmos)
 "lA" = (
@@ -13374,7 +13410,8 @@
 	},
 /obj/machinery/atmospherics/binary/pump/high_power{
 	dir = 1;
-	name = "CO2 to Hangar"
+	name = "CO2 to Hangar";
+	target_pressure = 15000
 	},
 /turf/simulated/floor/tiled/techfloor/grid,
 /area/quartermaster/shuttlefuel)
@@ -13680,14 +13717,12 @@
 /turf/simulated/floor/tiled/white,
 /area/shuttle/petrov/analysis)
 "Hm" = (
-/obj/machinery/portable_atmospherics/canister/carbon_dioxide/engine_setup{
-	start_pressure = 4769
-	},
 /obj/effect/floor_decal/industrial/outline/orange,
 /obj/effect/floor_decal/industrial/warning/corner{
 	dir = 8;
 	icon_state = "warningcorner"
 	},
+/obj/machinery/portable_atmospherics/canister/carbon_dioxide,
 /turf/simulated/floor/tiled/dark,
 /area/quartermaster/expedition/atmos)
 "Hn" = (
@@ -16613,9 +16648,7 @@
 	dir = 4;
 	icon_state = "map_connector"
 	},
-/obj/machinery/portable_atmospherics/canister/carbon_dioxide/engine_setup{
-	start_pressure = 7498
-	},
+/obj/machinery/portable_atmospherics/canister/carbon_dioxide,
 /turf/simulated/floor/plating,
 /area/guppy_hangar/start)
 "Re" = (
diff --git a/maps/torch/torch2_deck4.dmm b/maps/torch/torch2_deck4.dmm
index 2f27088846..e34a35e750 100644
--- a/maps/torch/torch2_deck4.dmm
+++ b/maps/torch/torch2_deck4.dmm
@@ -1199,7 +1199,7 @@
 /turf/simulated/floor/wood/mahogany,
 /area/command/captainmess)
 "dS" = (
-/turf/simulated/floor/carpet/blue2,
+/turf/simulated/floor/carpet/blue,
 /area/command/captainmess)
 "dT" = (
 /obj/effect/floor_decal/spline/fancy/wood{
@@ -1322,7 +1322,7 @@
 	dir = 8;
 	icon_state = "wooden_chair_wings_preview"
 	},
-/turf/simulated/floor/carpet/blue2,
+/turf/simulated/floor/carpet/blue,
 /area/command/captainmess)
 "en" = (
 /obj/machinery/conveyor{
@@ -1423,7 +1423,7 @@
 	dir = 4;
 	icon_state = "wooden_chair_wings_preview"
 	},
-/turf/simulated/floor/carpet/blue2,
+/turf/simulated/floor/carpet/blue,
 /area/command/captainmess)
 "eE" = (
 /obj/effect/floor_decal/industrial/warning{
@@ -1731,7 +1731,7 @@
 "fB" = (
 /obj/structure/table/woodentable/walnut,
 /obj/random_multi/single_item/runtime,
-/turf/simulated/floor/carpet/blue2,
+/turf/simulated/floor/carpet/blue,
 /area/command/captainmess)
 "fC" = (
 /obj/structure/cable/green{
@@ -2025,7 +2025,7 @@
 "gs" = (
 /obj/item/weapon/storage/box/donut,
 /obj/structure/table/woodentable/walnut,
-/turf/simulated/floor/carpet/blue2,
+/turf/simulated/floor/carpet/blue,
 /area/command/captainmess)
 "gw" = (
 /obj/structure/table/marble,
@@ -2293,7 +2293,7 @@
 	dir = 4
 	},
 /obj/structure/table/woodentable/walnut,
-/turf/simulated/floor/carpet/blue2,
+/turf/simulated/floor/carpet/blue,
 /area/command/captainmess)
 "hn" = (
 /obj/structure/bed/chair/wood/wings/mahogany{
@@ -2309,7 +2309,7 @@
 /obj/structure/disposalpipe/segment{
 	dir = 4
 	},
-/turf/simulated/floor/carpet/blue2,
+/turf/simulated/floor/carpet/blue,
 /area/command/captainmess)
 "ho" = (
 /obj/machinery/atmospherics/pipe/simple/hidden/supply{
@@ -3975,8 +3975,6 @@
 /area/maintenance/fourthdeck/foreport)
 "mM" = (
 /obj/effect/floor_decal/corner/blue/diagonal,
-/obj/item/weapon/ironingiron,
-/obj/item/weapon/ironingiron,
 /obj/item/weapon/storage/box/detergent,
 /obj/item/weapon/storage/box/detergent,
 /obj/structure/table/rack,
@@ -4370,6 +4368,7 @@
 	dir = 4;
 	pixel_x = -22
 	},
+/obj/item/weapon/ironingiron,
 /turf/simulated/floor/tiled/white,
 /area/crew_quarters/laundry)
 "nU" = (
@@ -4669,6 +4668,7 @@
 	pixel_x = -23;
 	pixel_y = 0
 	},
+/obj/item/weapon/ironingiron,
 /turf/simulated/floor/tiled/white,
 /area/crew_quarters/laundry)
 "pg" = (
@@ -12818,7 +12818,7 @@
 	pixel_y = 0
 	},
 /obj/structure/table/woodentable/walnut,
-/turf/simulated/floor/carpet/blue2,
+/turf/simulated/floor/carpet/blue,
 /area/command/captainmess)
 "OJ" = (
 /obj/machinery/computer/modular/preset/security{
@@ -16179,7 +16179,7 @@
 	dir = 4;
 	icon_state = "wooden_chair_wings_preview"
 	},
-/turf/simulated/floor/carpet/blue2,
+/turf/simulated/floor/carpet/blue,
 /area/command/captainmess)
 "Xh" = (
 /obj/structure/cable/green{
diff --git a/maps/torch/torch3_deck3.dmm b/maps/torch/torch3_deck3.dmm
index 9b40e74b68..7d58da4e6c 100644
--- a/maps/torch/torch3_deck3.dmm
+++ b/maps/torch/torch3_deck3.dmm
@@ -1396,7 +1396,7 @@
 "cT" = (
 /obj/structure/disposalpipe/segment,
 /obj/item/weapon/stool/bar/padded,
-/turf/simulated/floor/wood/walnut,
+/turf/simulated/floor/carpet/blue,
 /area/crew_quarters/mess)
 "cU" = (
 /obj/structure/table/standard,
@@ -1953,7 +1953,7 @@
 	},
 /obj/structure/disposalpipe/segment,
 /obj/item/weapon/stool/bar/padded,
-/turf/simulated/floor/wood/walnut,
+/turf/simulated/floor/carpet/blue,
 /area/crew_quarters/mess)
 "dZ" = (
 /obj/structure/cable/green{
@@ -3267,7 +3267,7 @@
 	dir = 4;
 	icon_state = "pipe-c"
 	},
-/turf/simulated/floor/wood/walnut,
+/turf/simulated/floor/carpet/blue,
 /area/crew_quarters/mess)
 "gw" = (
 /obj/effect/floor_decal/corner/grey/diagonal{
@@ -3298,11 +3298,11 @@
 	},
 /obj/machinery/atmospherics/pipe/simple/hidden/scrubbers,
 /obj/machinery/atmospherics/pipe/simple/hidden/supply,
-/turf/simulated/floor/wood/walnut,
+/turf/simulated/floor/carpet/blue,
 /area/crew_quarters/mess)
 "gz" = (
 /obj/structure/disposalpipe/junction/mirrored,
-/turf/simulated/floor/wood/walnut,
+/turf/simulated/floor/carpet/blue,
 /area/crew_quarters/mess)
 "gA" = (
 /obj/machinery/door/firedoor,
@@ -3783,7 +3783,7 @@
 /obj/machinery/atmospherics/pipe/simple/hidden/scrubbers,
 /obj/machinery/atmospherics/pipe/simple/hidden/supply,
 /obj/machinery/media/jukebox/old,
-/turf/simulated/floor/wood/walnut,
+/turf/simulated/floor/carpet/blue,
 /area/crew_quarters/mess)
 "hv" = (
 /obj/structure/railing/mapped{
@@ -3805,7 +3805,7 @@
 	d2 = 8;
 	icon_state = "4-8"
 	},
-/turf/simulated/floor/wood/walnut,
+/turf/simulated/floor/carpet/blue,
 /area/crew_quarters/mess)
 "hx" = (
 /obj/structure/fitness/weightlifter,
@@ -4323,7 +4323,7 @@
 /obj/machinery/door/firedoor,
 /obj/effect/floor_decal/industrial/hatch/yellow,
 /obj/effect/floor_decal/spline/fancy/wood/corner,
-/turf/simulated/floor/wood/walnut,
+/turf/simulated/floor/carpet/red,
 /area/crew_quarters/mess)
 "iw" = (
 /obj/effect/floor_decal/industrial/hatch/yellow,
@@ -5086,7 +5086,7 @@
 /obj/machinery/atmospherics/pipe/simple/hidden/supply{
 	dir = 6
 	},
-/turf/simulated/floor/wood/walnut,
+/turf/simulated/floor/carpet/blue,
 /area/crew_quarters/mess)
 "jW" = (
 /obj/machinery/atmospherics/pipe/simple/hidden/supply{
@@ -5103,7 +5103,7 @@
 /obj/machinery/atmospherics/pipe/simple/hidden/scrubbers{
 	dir = 4
 	},
-/turf/simulated/floor/wood/walnut,
+/turf/simulated/floor/carpet/blue,
 /area/crew_quarters/mess)
 "jX" = (
 /obj/effect/landmark{
@@ -5126,7 +5126,7 @@
 /obj/machinery/light/spot{
 	dir = 1
 	},
-/turf/simulated/floor/wood/walnut,
+/turf/simulated/floor/carpet/blue,
 /area/crew_quarters/mess)
 "jY" = (
 /obj/structure/disposalpipe/segment,
@@ -5135,7 +5135,7 @@
 	d2 = 8;
 	icon_state = "4-8"
 	},
-/turf/simulated/floor/wood/walnut,
+/turf/simulated/floor/carpet/blue,
 /area/crew_quarters/mess)
 "jZ" = (
 /turf/simulated/wall/r_wall/hull,
@@ -5152,7 +5152,7 @@
 	},
 /obj/machinery/atmospherics/pipe/simple/hidden/scrubbers,
 /obj/machinery/atmospherics/pipe/simple/hidden/supply,
-/turf/simulated/floor/wood/walnut,
+/turf/simulated/floor/carpet/blue,
 /area/crew_quarters/mess)
 "kc" = (
 /obj/machinery/atmospherics/pipe/simple/hidden/scrubbers{
@@ -5266,7 +5266,7 @@
 	dir = 2;
 	icon_state = "pipe-c"
 	},
-/turf/simulated/floor/wood/walnut,
+/turf/simulated/floor/carpet/blue,
 /area/crew_quarters/mess)
 "kn" = (
 /obj/item/device/radio/intercom/department/security{
@@ -5481,7 +5481,7 @@
 	icon_state = "1-2"
 	},
 /obj/machinery/atmospherics/pipe/simple/hidden/scrubbers,
-/turf/simulated/floor/wood/walnut,
+/turf/simulated/floor/carpet/blue,
 /area/crew_quarters/mess)
 "kR" = (
 /obj/structure/cable/green{
@@ -5779,7 +5779,7 @@
 "ly" = (
 /obj/structure/table/woodentable/walnut,
 /obj/structure/flora/pottedplant/deskfern,
-/turf/simulated/floor/wood/walnut,
+/turf/simulated/floor/carpet/blue,
 /area/crew_quarters/mess)
 "lA" = (
 /obj/structure/cable/green{
@@ -6460,14 +6460,14 @@
 	d2 = 2;
 	icon_state = "1-2"
 	},
-/turf/simulated/floor/wood/walnut,
+/turf/simulated/floor/carpet/blue,
 /area/crew_quarters/mess)
 "nj" = (
 /obj/machinery/atmospherics/unary/vent_pump/on{
 	dir = 8
 	},
 /obj/structure/disposalpipe/segment,
-/turf/simulated/floor/wood/walnut,
+/turf/simulated/floor/carpet/blue,
 /area/crew_quarters/mess)
 "nl" = (
 /obj/structure/bed/chair/office/dark{
@@ -7282,7 +7282,7 @@
 	name = "Bar";
 	sort_type = "Bar"
 	},
-/turf/simulated/floor/wood/walnut,
+/turf/simulated/floor/carpet/blue,
 /area/crew_quarters/mess)
 "pe" = (
 /obj/structure/cable/green{
@@ -7302,7 +7302,7 @@
 	dir = 4
 	},
 /obj/structure/disposalpipe/segment,
-/turf/simulated/floor/wood/walnut,
+/turf/simulated/floor/carpet/blue,
 /area/crew_quarters/mess)
 "pf" = (
 /obj/machinery/firealarm{
@@ -7326,7 +7326,7 @@
 	c_tag = "Mess Hall - Port";
 	dir = 1
 	},
-/turf/simulated/floor/wood/walnut,
+/turf/simulated/floor/carpet/blue,
 /area/crew_quarters/mess)
 "pj" = (
 /obj/structure/cable/green{
@@ -8290,7 +8290,7 @@
 /obj/machinery/door/firedoor,
 /obj/effect/floor_decal/industrial/hatch/yellow,
 /obj/effect/floor_decal/spline/fancy/wood,
-/turf/simulated/floor/wood/walnut,
+/turf/simulated/floor/carpet/red,
 /area/crew_quarters/mess)
 "rM" = (
 /obj/machinery/atmospherics/pipe/simple/hidden/supply{
@@ -8304,7 +8304,7 @@
 	d2 = 8;
 	icon_state = "4-8"
 	},
-/turf/simulated/floor/wood/walnut,
+/turf/simulated/floor/carpet/blue,
 /area/crew_quarters/mess)
 "rP" = (
 /obj/machinery/atmospherics/pipe/manifold/hidden/supply{
@@ -9117,7 +9117,7 @@
 /area/hallway/primary/thirddeck/aft)
 "sY" = (
 /obj/structure/disposalpipe/segment,
-/turf/simulated/floor/wood/walnut,
+/turf/simulated/floor/carpet/blue,
 /area/crew_quarters/mess)
 "sZ" = (
 /obj/machinery/atmospherics/pipe/simple/hidden/supply,
@@ -11780,7 +11780,7 @@
 	dir = 4;
 	pixel_x = 24
 	},
-/turf/simulated/floor/wood/walnut,
+/turf/simulated/floor/carpet/blue,
 /area/crew_quarters/mess)
 "Ar" = (
 /obj/machinery/atmospherics/unary/vent_scrubber/on{
@@ -11847,7 +11847,7 @@
 /obj/structure/disposalpipe/segment{
 	dir = 4
 	},
-/turf/simulated/floor/wood/walnut,
+/turf/simulated/floor/carpet/blue,
 /area/crew_quarters/mess)
 "AB" = (
 /obj/structure/cable/green{
@@ -11868,7 +11868,7 @@
 	dir = 1
 	},
 /obj/machinery/vending/cigarette,
-/turf/simulated/floor/carpet/red,
+/turf/simulated/floor/carpet/blue,
 /area/crew_quarters/mess)
 "AD" = (
 /obj/structure/table/standard,
@@ -13388,7 +13388,7 @@
 	},
 /obj/machinery/disposal,
 /obj/effect/floor_decal/industrial/hatch/yellow,
-/turf/simulated/floor/wood/walnut,
+/turf/simulated/floor/carpet/red,
 /area/crew_quarters/mess)
 "Ej" = (
 /obj/machinery/door/morgue{
@@ -16146,7 +16146,7 @@
 /obj/machinery/atmospherics/pipe/simple/hidden/supply{
 	dir = 4
 	},
-/turf/simulated/floor/wood/walnut,
+/turf/simulated/floor/carpet/blue,
 /area/crew_quarters/mess)
 "Kp" = (
 /obj/machinery/power/apc{
@@ -16672,7 +16672,7 @@
 	dir = 8;
 	icon_state = "pipe-c"
 	},
-/turf/simulated/floor/wood/walnut,
+/turf/simulated/floor/carpet/blue,
 /area/crew_quarters/mess)
 "Ly" = (
 /obj/effect/floor_decal/corner/green{
@@ -18307,7 +18307,7 @@
 	d2 = 2;
 	icon_state = "1-2"
 	},
-/turf/simulated/floor/wood/walnut,
+/turf/simulated/floor/carpet/blue,
 /area/crew_quarters/mess)
 "PP" = (
 /obj/structure/disposalpipe/segment,
@@ -19299,7 +19299,7 @@
 	icon_state = "1-2"
 	},
 /obj/machinery/hologram/holopad,
-/turf/simulated/floor/wood/walnut,
+/turf/simulated/floor/carpet/blue,
 /area/crew_quarters/mess)
 "Sw" = (
 /obj/structure/sign/hydro,
@@ -19487,7 +19487,7 @@
 	d2 = 4;
 	icon_state = "2-4"
 	},
-/turf/simulated/floor/wood/walnut,
+/turf/simulated/floor/carpet/blue,
 /area/crew_quarters/mess)
 "ST" = (
 /obj/machinery/atmospherics/pipe/simple/hidden/scrubbers{
@@ -19527,7 +19527,7 @@
 	icon_state = "pipe-c"
 	},
 /obj/machinery/vending/coffee,
-/turf/simulated/floor/carpet/red,
+/turf/simulated/floor/carpet/blue,
 /area/crew_quarters/mess)
 "Tc" = (
 /obj/machinery/atmospherics/unary/vent_pump/on,
@@ -19739,7 +19739,7 @@
 	d2 = 8;
 	icon_state = "4-8"
 	},
-/turf/simulated/floor/wood/walnut,
+/turf/simulated/floor/carpet/blue,
 /area/crew_quarters/mess)
 "Ty" = (
 /obj/item/weapon/stool/padded,
@@ -20198,7 +20198,7 @@
 /obj/machinery/atmospherics/pipe/simple/hidden/scrubbers{
 	dir = 9
 	},
-/turf/simulated/floor/wood/walnut,
+/turf/simulated/floor/carpet/blue,
 /area/crew_quarters/mess)
 "UL" = (
 /obj/effect/floor_decal/techfloor{
@@ -20706,7 +20706,7 @@
 	name = "Hydroponics";
 	sort_type = "Hydroponics"
 	},
-/turf/simulated/floor/wood/walnut,
+/turf/simulated/floor/carpet/blue,
 /area/crew_quarters/mess)
 "VU" = (
 /turf/simulated/floor/tiled,
@@ -20958,7 +20958,7 @@
 	dir = 4;
 	icon_state = "pipe-c"
 	},
-/turf/simulated/floor/wood/walnut,
+/turf/simulated/floor/carpet/blue,
 /area/crew_quarters/mess)
 "WI" = (
 /obj/effect/floor_decal/corner/green{
@@ -21210,7 +21210,7 @@
 /turf/simulated/floor/reinforced/airless,
 /area/space)
 "Xs" = (
-/turf/simulated/floor/wood/walnut,
+/turf/simulated/floor/carpet/blue,
 /area/crew_quarters/mess)
 "Xt" = (
 /obj/machinery/meter,
@@ -22092,7 +22092,7 @@
 	pixel_x = 24
 	},
 /obj/machinery/vending/mredispenser,
-/turf/simulated/floor/carpet/red,
+/turf/simulated/floor/carpet/blue,
 /area/crew_quarters/mess)
 "ZL" = (
 /obj/effect/floor_decal/spline/fancy/black{
@@ -22118,7 +22118,7 @@
 	dir = 4
 	},
 /obj/structure/flora/pottedplant/stoutbush,
-/turf/simulated/floor/carpet/red,
+/turf/simulated/floor/carpet/blue,
 /area/crew_quarters/mess)
 "ZR" = (
 /obj/machinery/atmospherics/unary/outlet_injector{
@@ -37823,7 +37823,7 @@ kc
 dA
 mF
 fp
-fp
+oH
 oH
 il
 ZB
@@ -40454,7 +40454,7 @@ gA
 fB
 Ei
 Yf
-fU
+Xs
 ZQ
 Xs
 Xs
@@ -40659,8 +40659,8 @@ nD
 ct
 nD
 vz
-fU
-fU
+vz
+vz
 vz
 fB
 tF
diff --git a/maps/torch/torch4_deck2.dmm b/maps/torch/torch4_deck2.dmm
index faeb7ae765..d2c17c49d0 100644
--- a/maps/torch/torch4_deck2.dmm
+++ b/maps/torch/torch4_deck2.dmm
@@ -8832,10 +8832,8 @@
 "tP" = (
 /obj/machinery/atmospherics/binary/passive_gate/on{
 	dir = 8;
-	max_pressure_setting = 15000;
 	name = "CO2 to Hangar";
-	regulate_mode = 2;
-	target_pressure = 2500;
+	target_pressure = 15000;
 	use_power = 1
 	},
 /obj/machinery/camera/network/engineering{
diff --git a/maps/torch/torch5_deck1.dmm b/maps/torch/torch5_deck1.dmm
index 010bf14b0a..42b1156425 100644
--- a/maps/torch/torch5_deck1.dmm
+++ b/maps/torch/torch5_deck1.dmm
@@ -24363,12 +24363,7 @@
 /area/operations_storage/medical)
 "mVb" = (
 /obj/machinery/nuclearbomb/station{
-	light_falloff_curve = 10;
-	maxTime = 600;
-	minTime = 600;
-	name = "ship self-destruct terminal";
-	self_destruct_cutoff = 300;
-	timeleft = 600
+	name = "ship self-destruct terminal"
 	},
 /turf/simulated/floor/blackgrid,
 /area/security/nuke_storage)
diff --git a/maps/torch/torch6_bridge.dmm b/maps/torch/torch6_bridge.dmm
index ee91837a5d..88cf07855e 100644
--- a/maps/torch/torch6_bridge.dmm
+++ b/maps/torch/torch6_bridge.dmm
@@ -5145,6 +5145,7 @@
 	dir = 5
 	},
 /obj/effect/floor_decal/spline/plain/beige,
+/obj/structure/closet/walllocker,
 /turf/simulated/floor/tiled/dark,
 /area/crew_quarters/heads/office/sea)
 "kw" = (
@@ -9609,6 +9610,7 @@
 	},
 /obj/item/weapon/pen,
 /obj/item/weapon/material/clipboard,
+/obj/item/weapon/storage/secure/briefcase/nukedisk,
 /turf/simulated/floor/wood/walnut,
 /area/crew_quarters/heads/office/sea)
 "vl" = (
@@ -16028,7 +16030,6 @@
 	pixel_y = -28;
 	req_access = list(list("ACCESS_ENGINE_EQUIP","ACCESS_ATMOS"))
 	},
-/obj/item/weapon/storage/secure/briefcase/nukedisk,
 /turf/simulated/floor/wood/walnut,
 /area/crew_quarters/heads/cobed)
 "NE" = (
@@ -17980,6 +17981,14 @@
 	dir = 8;
 	icon_state = "techfloor_edges"
 	},
+/obj/structure/closet/cabinet{
+	anchored = 1;
+	name = "Delta Alert Cabinet";
+	pixel_x = -25;
+	req_access = list("ACCESS_VAULT");
+	wall_mounted = 1
+	},
+/obj/item/weapon/storage/secure/briefcase/nukedisk,
 /turf/simulated/floor/tiled/techfloor,
 /area/crew_quarters/safe_room/bridge)
 "To" = (
@@ -18084,6 +18093,10 @@
 /obj/item/device/radio/phone,
 /turf/simulated/floor/tiled/dark,
 /area/turret_protected/ai_outer_chamber)
+"TG" = (
+/obj/structure/closet/walllocker,
+/turf/simulated/wall/r_wall/prepainted,
+/area/crew_quarters/safe_room/bridge)
 "TI" = (
 /obj/machinery/atmospherics/pipe/simple/hidden/supply{
 	dir = 5
@@ -30188,7 +30201,7 @@ ti
 ti
 PH
 ti
-ti
+TG
 ti
 ti
 ti
-- 
2.30.2.windows.1

